addons/im_livechat/models/discuss_channel.py

```diff
@@ -352,6 +352,19 @@ def _email_livechat_transcript(self, email):
             'body_html': mail_body,
         })
         mail.send()
+        if (
+            not self.env.user._is_public()
+            and not self.livechat_customer_partner_ids.filtered(
+                lambda l: l.id == self.env.user.partner_id.id
+            )
```

Comment:
```suggestion
            self.env.user.partner_id in  self.livechat_agent_partner_ids
```

It's easier to confirm that the current user is an agent. But do we really want to send the message only in that case? Why not always send it?

Reply:
According to spec, the message should only be posted when an agent sends the conversation, not when a visitor sends it to themselves.

Reply:
I don't see anything about restricting this message to agents on the task.

Reply:
here is the link of the [message](https://www.odoo.com/mail/message/724533983).

Reply:
Fine then, lets only simplify the condition :)